plays/server-config.yaml

```diff
@@ -10,6 +10,6 @@
       tags:
         - ssl
         - sslrenew
-    - role: ufw_rules
-      tags:
-        - ufw
+    # - role: ufw_rules
```

Comment:
Please reinclude the ufw_rules. :-) If someone don't need it, they can skip it by "--skip-tags".

Reply:
That's right! uncommenting it soon.